rust-checks/src/bin/no-python-tests.rs

```diff
@@ -0,0 +1,33 @@
+use nixpkgs_hammering_ast_checks::analysis::*;
+use nixpkgs_hammering_ast_checks::common_structs::*;
+use regex::Regex;
+use std::error::Error;
+use std::io::{self, BufRead, BufReader};
+use std::process::ChildStdout;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    let attrs: Vec<Attr> = serde_json::from_reader(io::stdin())?;
+    println!("{}", analyze_log_files(attrs, analyze_single_file)?);
+    Ok(())
+}
+
+fn analyze_single_file(log: BufReader<ChildStdout>) -> Result<Report, Box<dyn Error>> {
+    let re = Regex::new(r"Ran 0 tests in 0.000s").unwrap();
```

Comment:
I wonder if the test runner runs timer within tests only. Otherwise 0.001s could possibly happen.

Reply:
I don't think I've ever seen "0 tests in 0.001s". @SuperSandro2000, have you? Obviously this depends to some degree on the internal details of pytest, but I think it's the standard behavior.

Reply:
I should have just searched. [Apparently](https://duckduckgo.com/?t=canonical&q=%220+tests+in+0.001s%22&ia=web), it can happen.

Reply:
I'll change the regex to "Ran 0 tests in \d+.\d+s"?